src/test/java/org/fcrepo/storage/ocfl/DefaultOcflObjectSessionTest.java

```diff
@@ -619,7 +622,7 @@ public void listAgVersions() {
 
         assertThat(session3.listVersions(DEFAULT_AG_ID).stream()
                 .map(OcflVersionInfo::getVersionNumber)
-                .collect(Collectors.toList()), contains("v1"));
+                .collect(Collectors.toList()), contains("v1", "v2", "v3"));
```

Comment:
These are new versions. The original code did not version the AG when an AG part was versioned. One of the changes in this PR is to change the behavior so that AGs are now versioned when their parts are versioned.